addons/hr_expense/models/hr_expense.py

```diff
@@ -217,17 +243,39 @@ def _compute_is_ref_editable(self):
             else:
                 expense.is_ref_editable = is_account_manager
 
+    @api.depends_context('lang')
+    @api.depends('product_id')
+    def _compute_product_description(self):
```

Comment:
Not entirely sure I understand the need for this.
Seems like the field product_description could just be a simple related towards product_id.description. Field is readonly and only used in the view as information purpose (or so it seems), I guess all that code using html_sanitize and is_not_html_empty which might downgrade performance is only used to hide a label in case we have a description that only contains an empty <span>. If this is indeed the reason, why not just use a related as I said and if we have an empty span, I don't really mind having an empty label showing. Seems like too much complexity for a small usability detail;

Reply:
@csnauwaert we could have used a related but then, we have an ugly empty line between the product & the price. This is just for a UX tweeking. It's only 6 lines of code, for a better design in the view (it's strange to have an empty extra line)

Reply:
I see your point and indeed, that's why all the complexity to not show empty space. I guess it's give and take. In line with the task PO, we will keep it. But thanks for raising a concern and for your feedback :)